src/components/App/App.js

```diff
@@ -65,7 +65,7 @@ class App extends Component {
     this.setState({
       name,
     });
-    report('Filter', 'name', 'name');
+    report('Filter', 'name', name);
```

Comment:
Thanks for the thought but we're not sending the name in purpose because we can't delete that data when user delete himself from our system.